packages/ui/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "@ark-market/tsconfig/internal-package.json",
+  // "extends": "@ark-market/tsconfig/internal-package.json",
```

Comment:
Is this commented for a specific reason, if not we should keep this tsconfig as it was.